opentreemap/treemap/units.py

```diff
@@ -98,6 +98,9 @@ def convert_to_database_units(self):
 }
 _unit_conversions["ft"] = {u: v * 12 for (u, v)
                            in _unit_conversions["in"].iteritems()}
+_unit_conversions["sq_ft"] = {
+    u: v / _unit_conversions["sq_m"]["sq_ft"]
+    for u, v in _unit_conversions["sq_m"].iteritems()}
```

Comment:
Only if we switch both drainage_area and greenInfrastructure.area, and I'm very reluctant to switch the latter.